docker/run.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /bin/bash 
```

Comment:
Is this change needed/meaningful? If not, would you mind reverting it?

The same applies to `docker/wait.sh` and `scripts/docker`